src/CoreWCF.Kafka/tests/Helpers/DockerEx.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Threading.Tasks;
+using Xunit.Abstractions;
+
+namespace CoreWCF.Kafka.Tests.Helpers;
+
+public static class DockerEx
+{
+    public static async Task PauseAsync(ITestOutputHelper output, string containerName)
+    {
```

Comment:
Would it be worth using [Docker.DotNet](https://www.nuget.org/packages/Docker.DotNet/)?